src/app/containers/App/index.test.jsx

```diff
@@ -3,36 +3,68 @@
  * https://github.com/jtart/react-universal-app
  */
 import React from 'react';
-import { shouldShallowMatchSnapshot } from '../../../testHelpers';
+import { render } from '@testing-library/react';
+import { StaticRouter, BrowserRouter } from 'react-router-dom';
 import { ClientApp, ServerApp } from '.';
+// eslint-disable-next-line import/no-named-as-default
+import App from './App';
+
+jest.mock('./App', () => jest.fn(() => <div />));
+
+jest.mock('react-router-dom', () => ({
+  BrowserRouter: jest.fn(),
+  StaticRouter: jest.fn(),
+}));
+BrowserRouter.mockImplementation(({ children }) => <div>{children}</div>);
+StaticRouter.mockImplementation(({ children }) => <div>{children}</div>);
 
 describe('ClientApp', () => {
-  shouldShallowMatchSnapshot(
-    'should render correctly',
-    <ClientApp data="someData!" routes={['someRoute']} />,
-  );
+  it('should render correctly', () => {
+    render(<ClientApp data="someData!" routes={['someRoute']} />);
+    expect(App).toHaveBeenCalledWith(
+      { initialData: 'someData!', routes: ['someRoute'] },
+      {},
+    );
+    expect(BrowserRouter).toHaveBeenCalledWith(
+      {
+        children: expect.anything(),
```

Comment:
We need to find an alternative to `expect.anything()`, happy to help find one :)